test/integration/narrow/routes/capture.test.js

```diff
@@ -0,0 +1,81 @@
+describe('Capture test', () => {
+  jest.mock('ffc-messaging')
+  jest.mock('../../../../app/plugins/crumb')
+  const createServer = require('../../../../app/server')
+  let server
+  const url = '/capture'
+
+  jest.mock('../../../../app/debt')
+  const { getDebts } = require('../../../../app/debt')
+
+  const debts = [{
+    scheme: 'SFI Pilot',
+    frn: '1234567890',
+    reference: 'SFIP1234567',
+    netValue: 1000.00,
+    debtType: 'Irregular',
+    recoveryDate: '19/01/2022',
+    attachedDate: '',
+    createdBy: 'John Watson'
+  },
+  {
+    scheme: 'SFI',
+    frn: '1234567891',
+    reference: 'SFIP1234568',
+    netValue: 570.00,
+    debtType: 'Administrative',
+    recoveryDate: '18/01/2022',
+    attachedDate: '18/01/2022',
+    createdBy: 'Steve Dickinson'
+  }]
+
+  beforeEach(async () => {
+    getDebts.mockResolvedValue(debts)
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterEach(async () => {
+    jest.resetAllMocks()
+    await server.stop()
+  })
+
+  describe('GET requests', () => {
+    const method = 'GET'
+
+    test('GET /capture route returns 200', async () => {
+      const options = {
+        method,
+        url
+      }
+
+      const response = await server.inject(options)
+      expect(response.statusCode).toBe(200)
+      expect(response.request.response.variety).toBe('view')
+      expect(response.request.response.source.template).toBe('capture')
+    })
+  })
+
+  describe('POST requests', () => {
+    const method = 'POST'
+
+    test.each([
```

Comment:
I dig this, would love to hear Edi/Kaz's opinion but as a developer, this saves so much boilerplate code making it easier to write, debug and review (as I'm doing now)

Something I would encourage (but want to hear Edi/Kaz's opinion too)

Reply:
Cool